src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
+import './assets/styles/main.css';
 
 function App() {
```

Comment:
* Kindly make use of class-based components as it is required for this project [Refer to the project requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components.)